include/rtc/rtptwcchandler.hpp

```diff
@@ -0,0 +1,25 @@
+#ifndef RTC_RTP_TWCC_HANDLER_H
+#define RTC_RTP_TWCC_HANDLER_H
+
+#if RTC_ENABLE_MEDIA
+
+#include "mediahandler.hpp"
+#include "rtp.hpp"
+namespace rtc {
+
+class RTC_CPP_EXPORT TwccHandler final : public MediaHandler {
+	RtpTwccExt twccHeader;
+	uint16_t twccSeqNum;
+	// Callback below is for recording TWCC seqNums
+	std::function<void(message_vector &)> processPacketsCallback;
```

Comment:
Why do you need a callback? The user should not be responsible for this, if you want to record the sequence number you should pass the recording object and call the corresponding method directly from the handler.